improve English (#457)
TYPO3-Documentation/TYPO3CMS-Book-ExtbaseFluid: Documentation/5-Domain/2-implementing-the-domain-model.rst

By default, Extbase invites all child objects with the parent object (so for
example, all offers of an organization). This behavior is called Eager-Loading.

> The annotation :php:`@TYPO3\CMS\Extbase\Annotation\ORM\Lazy` causes Extbase to load and build the objects only when they
> are actually needed (lazy loading). This can be appropriate for big data records,
> e.g., many organizations, each with very many offers. That can lead to a significant
> increase in speed.

Reviewer: ```suggestion
The annotation :php:`@TYPO3\CMS\Extbase\Annotation\ORM\Lazy` causes Extbase to load and build the objects only when they
are actually needed (lazy loading). This can lead to a significant
increase in speed in scenarios with big data records,
for example, many organizations, each with many offers. 
```

are actually needed (lazy loading). This can be appropriate for big data records,
e.g., many organizations, each with very many offers. That can lead to a significant
increase in speed.

.. note::

   Beware, however, against all the properties provided by child objects with

>    :php:`@TYPO3\CMS\Extbase\Annotation\ORM\Lazy`, because this can lead to frequent loading of child objects. The ensuing,

Reviewer: I think it's important to keep the "because".

>    The ensuing small-scaled database accesses reduce the performance and will cause the exact

Reviewer: ```suggestion
   The ensuing small-scale accesses of the database reduce the performance and will cause the exact
```

> The annotation :php:`@TYPO3\CMS\Extbase\Annotation\ORM\Cascade("remove")` causes if the organization is deleted,
> the offers will also be deleted immediately. Extbase usually leaves all
> child objects persist unchanged.

Reviewer: ```suggestion
The annotation :php:`@TYPO3\CMS\Extbase\Annotation\ORM\Cascade("remove")` means that if the organization is deleted,
the offers will also be deleted immediately. Extbase usually leaves all
child objects' persistence unchanged.
```
